resources/v2/02_subjects.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE subjects
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    name text NOT NULL,
+    kind text NOT NULL,
```

Comment:
What is kind here with subjects ?

Reply:
`kind` refers to whether it's related to `social science` or `environmental science` or `psychology`.

Reply:
Lets say subject name is Science. Now what will you add in kind ?